README.md

```diff
@@ -1,10 +1,20 @@
 # AzerothShard module
 
-**⚠️ DISCLAIMER ⚠️** : this module is a **work in progress**, hence is not intended to be distributed yet.
+> [!IMPORTANT]  
+> AzerothShard has released their collection of modules to the public and no longer being maintained.
 
-This module is a collection of custom features that have been implemented privately on AzerothShard project.
-Finally, most of them will be released open-source using an all-in-one module since they are coupled between eachother
-Although possibly in future this module will be refactorized to create standalone modules for each feature.
+## Modules that have been exported to AzerothCore
+
+- `mod-arena-solo-3v3` => [mod-arena-3v3-solo-queue](https://github.com/azerothcore/mod-arena-3v3-solo-queue) 
+- `mod-xp-rates` => [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
```

Comment:
not sure if these modules are the same, moreover I think that we are missing some modules in the list

Reply:
I'm not sure https://github.com/azerothcore/mod-chromie-xp is exactly the same. The xp mod from CC was also integrated into the smarstone. But maybe we can still add the mod-chromie-xp as a possible replacement although it's not 1:1

Btw, I'm not sure if I understood the question fully